aspnet/getting-started/choosing-the-right-dotnet.rst

```diff
@@ -1,37 +1,65 @@
 Choosing the Right .NET For You on the Server
 =============================================
 
-By `Daniel Roth`_
+By `Daniel Roth`_ and `Rick Anderson`_
 
-ASP.NET 5 is based on the :doc:`.NET Execution Environment (DNX) </dnx/overview>`, which supports running cross-platform on Windows, Mac and Linux. When selecting a DNX to use you also have a choice of .NET flavors to pick from: .NET Framework (CLR), :doc:`.NET Core </conceptual-overview/dotnetcore>` (CoreCLR) or `Mono <http://mono-project.com>`_. Which .NET flavor should you choose? Let's look at the pros and cons of each one.
+ASP.NET 5 is based on the :doc:`.NET Execution Environment (DNX) </dnx/overview>`, which supports running cross-platform on Windows, Mac and Linux. DNX come in three .NET versions: 
+
+- .NET Framework (Also known as the "*Full*" .NET Framework)
```

Comment:
I want to just remove the comment. The official terms are .NET Framework and .NET Core.